_includes/footer.html

```diff
@@ -84,7 +84,8 @@
         <div class="bottom_footer">
             <div class="footer-copyright col-xs-12 col-md-8">
                 <p class="copyright">
-                    Copyright &copy; 2013-2021 Docker Inc. All rights reserved. </p>
+                    Copyright <i class="fa-regular fa-copyright" aria-hidden="true"></i> 2013-{{ site.time | date: '%Y' }} Docker Inc. All rights reserved.
```

Comment:
curious why you picked a fontAwesome font for this one (as `©` should be fine I think?)

Reply:
just avoid unicode in our code base but I'm ok to bring this back up.